### PR TITLE
Feature/cockpit demo 2021 dunfell

### PR DESCRIPTION
**Pull request by iusyk**
Update the manifests to support the cockpit-demo based at dunfell

### CODE REVIEW

**Comment by rshym** [comment]
*504e2d Cockpit-demp-2021: fit manifest to teh actual releases*
- Commit message - misprint "teh"
- Commit message - Use spaces before dash (-)
- Commit message - "no current tag is" ? Should "no" be replaced by "now"?
- Change of dom0.xml is removed by commit "Dom0: shift dom0 manifest to dunfell"
- Change of domd.xml is implicitly reverted by commit "DomD: change at dunfell"
So this path is not needed.

*cded1f DomD: add agl-profile-cluster-qt5*
*f95d67 Revert "DomD: add agl-profile-cluster-qt5"*
These two patches are not needed.

*574e9f Cockpit-demo: remove meta-agl-extra from domd.xml*
Commit message - misprint "teh"

*b16970 DomD: change at dunfell*
Commit message - use "to" instead of "at"

*fa5912 Cockpit-demo: add meta-python2*
Empty line 41

*5975d5 Dom0: shift dom0 manifest to dunfell*
No comments.

**Comment by iusyk** [comment]
> _504e2d Cockpit-demp-2021: fit manifest to teh actual releases_
> 
>     * Commit message - misprint "teh"
> 
>     * Commit message - Use spaces before dash (-)
> 
>     * Commit message - "no current tag is" ? Should "no" be replaced by "now"?
> 
>     * Change of dom0.xml is removed by commit "Dom0: shift dom0 manifest to dunfell"
> 
>     * Change of domd.xml is implicitly reverted by commit "DomD: change at dunfell"
>       So this path is not needed.
> 
> 
> _cded1f DomD: add agl-profile-cluster-qt5_
> _f95d67 Revert "DomD: add agl-profile-cluster-qt5"_
> These two patches are not needed.
> 
> _574e9f Cockpit-demo: remove meta-agl-extra from domd.xml_
> Commit message - misprint "teh"
> 
> _b16970 DomD: change at dunfell_
> Commit message - use "to" instead of "at"
> 
> _fa5912 Cockpit-demo: add meta-python2_
> Empty line 41
> 
> _5975d5 Dom0: shift dom0 manifest to dunfell_
> No comments.

fixed

File: prod_cockpit_demo/domd.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <remote name="epam" fetch="ssh://git@gitpct.epam.com" />
 
   <!-- defaults -->
-  <default remote="agl" sync-j="4" revision="refs/tags/icefish/9.0.2"/>
+  <default remote="agl" sync-j="4" revision="refs/tags/koi/11.0.1"/>
```
Comment (rshym) [inline prod_cockpit_demo/domd.xml]: Pay attention, that this change is commented out in following commit "DomD: change to dunfell".
https://github.com/xen-troops/manifests/pull/88/commits/b3474cbd5b2507f3f14df4e18aff78742562fc98#diff-8633a38636ea7c613baeffead3c390ea6eaca7d41c6f67d843d1e742d2811629R16

If it is intentionally then I'm OK.

File: prod_cockpit_demo/dom0.xml [hunk 2]
```diff
@@ -10,5 +10,5 @@
     <project remote="yoctoproject" name="meta-virtualization" path="meta-virtualization" upstream="thud" revision="9e8c0c96b443828a255e7d6ca6291598347672ac" />
     <project remote="yoctoproject" name="poky" path="poky" upstream="thud" revision="e7f0177ef3b6e06b8bc1722fca0241fef08a1530" />
     <project remote="openembedded" name="meta-openembedded" path="meta-openembedded" upstream="thud" revision="4cd3a39f22a2712bfa8fc657d09fe2c7765a4005" />
-    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="thud" revision="b30036d4ef7ce9fe746833fc54de6ac7b0e00638" />
+    <project remote="linaro" name="openembedded/meta-linaro" path="meta-linaro" upstream="thud" revision="0a94decea3bd2504590d1637eadff9d502c19ee2" />
```
Comment (rshym) [inline prod_cockpit_demo/dom0.xml]: Do we need this change?
This change is removed in following commit "Dom0: shift dom0 manifest to dunfell":
https://github.com/xen-troops/manifests/pull/88/commits/a112a529fd6ff65bb4a00b0de6e3f6f64bd62183#diff-c2246cac11b492d8a68de8a6bdd86e083e164bab1d70999b0de5df9d55cb2f9dL13

**Review by rshym — COMMENTED** [review]
*504e2d Cockpit-demp-2021: fit manifest to teh actual releases*
- Commit message - remove 'i' after 'dom0'
- Taking into account that changes made in this commit are removed (and commented out) by following commits, I's good to check - is this commit needed.

*574e9f Cockpit-demo: remove meta-agl-extra from domd.xml*
Commit message - misprint "teh"

**Pull request closed without merging**